contents/python/introduction-to-python.md

```diff
@@ -116,4 +108,11 @@ Learning how to avoid such pitfalls is one huge step towards being a productive
 
 * [Top 10 Common Mistakes of Python Programmers](https://www.toptal.com/python/top-10-mistakes-that-python-programmers-make)
 * [Common Python Gotchas](https://sopython.com/wiki/Common_Gotchas_In_Python)
+
+## A Note on Python 2
```

Comment:
Sometimes readers don't read all the sections and just focus on sections they want to read. Better to name it - "Python 2 discontinued" or something along those lines? So that even if the reader doesn't read the section, by seeing the heading, he at least kind of gets an idea.

Reply:
That's a good suggestion. Will change

Reply:
I've changed the title to `Python 2: a Discontinued Version`

Reply:
@Tejas2805, thanks a lot for taking he time to review this. 
Do you think the new title is suitable?